pages/_app.js

```diff
@@ -4,7 +4,7 @@ import CustomHead from "./CustomHead";
 import "../styles/custom.scss";
 
 const ISSUES_ENDPOINT =
-  "https://data.austintexas.gov/resource/rzwg-fyv8.json?$limit=10000";
+  "https://data.austintexas.gov/resource/rzwg-fyv8.json?$limit=100000";
```

Comment:
A burley patron enters the bar, slams the counter and says in a commanding voice: I want 10 times more beer than everyone here is having.

The bartender says: Now thats an order of magnitude!

Reply:
🥁 ba dum sh!